rllib/utils/test_utils.py

```diff
@@ -818,9 +818,9 @@ def framework_iterator(
             continue
         # Skip if tf/tf2 and py >= 3.11.
         elif fw in ["tf", "tf2"] and (
-            sys.version_info.major == 3 and sys.version_info.minor >= 11
+            sys.version_info.major == 3 and sys.version_info.minor >= 9
```

Comment:
Why did we lower this to 3.9? B/c of the parallel upgrade of the tf version?

Reply:
yess, sorry, i pinged in the other PR; tests on CI are still running with python 3.9; but build the ray docker image across python 3.9-3.11;

we need to upgrade tensorflow to build the docker images for python 3.11; but the upgrade also break these tests for python 3.9 which is the version CI tests using